save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/cpg/CpgGraph.kt

```diff
@@ -91,16 +91,8 @@ data class CpgGraph(
          * Placeholder of a graph
          */
         val placeholder = CpgGraph(
-            nodes = listOf(
-                CpgNode("1", CpgNodeAttributes("Alisson", "#FF0000")),
-                CpgNode("2", CpgNodeAttributes("John", "#00FF00")),
-                CpgNode("3", CpgNodeAttributes("Sam", "#0000FF")),
-            ),
-            edges = listOf(
-                CpgEdge("1->2", "1", "2", CpgEdgeAttributes("KNOWS")),
-                CpgEdge("2->3", "2", "3", CpgEdgeAttributes("LIKES")),
-                CpgEdge("3->1", "3", "1", CpgEdgeAttributes("HATES")),
-            ),
+            nodes = emptyList(),
+            edges = emptyList(),
```

Comment:
Probably we need to generate once the graph for code placeholder and put it here instead of an empty graph

Reply:
I do not get it. 

We work with a stub:
```
try {
     buildGraph
} catch {
     return stub
}
```

Reply:
Ah, you want to create some placeholder graph?

Reply:
> Ah, you want to create some placeholder graph? For some placeholder code like "hello world"?

Yes, definitely

Reply:
Better to have crying cat instead of a stub :)